apps/aggiemap-angular/src/environments/notification-events.ts

```diff
@@ -59,5 +59,14 @@ export const NotificationEvents: NotificationProperties[] = [
     message: 'Thank you! Your feedback has been submitted successfully and the Aggiemap team notified.',
     imgUrl: './assets/images/like.svg',
     imgAltText: 'Feedback Icon'
+  },
+  {
```

Comment:
At the beginning of `notification-events.ts` you have a commented out object, you saving that for a rainy day?

Reply:
Commented it in and removed the active date range and will only show when manually invoked.